syntax= "proto3";

service Notification{
  rpc Notif1(Notif) returns (void2){};

}

message void2{

}
message Notif{
  string message=1;
}